manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu

```

```suggestion
PackageIdentifier: LibNyanpasu.clash-nyanpasu
```
`Publisher.Package` format is preferred for GitHub releases